index.php

```diff
@@ -0,0 +1,30 @@
+<?php include('connect.php'); ?><!DOCTYPE html>
```

Comment:
okay true, i need a better way to compile php from jade because it doesn't seem to work too well